draft-packages/form/KaizenDraft/Form/Primitives/FieldMessage/styles.scss

```diff
@@ -5,8 +5,9 @@
 @import "../../decision-tokens.scss";
 
 .message {
-  @include kz-typography-paragraph-small;
-  @include ca-inherit-baseline;
+  display: flex;
+  align-items: center;
+  gap: $ca-grid * 0.25;
```

Comment:
We have replaced `@include kz-typography-paragraph-small;` with a use of the `Paragraph `component which defines the typography.